### PR TITLE
Wrong datas in order return with multiple products

### LINKED ISSUE #29839: [FO] Wrong datas in order return with multiple products
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

The data is incorrect when making a return with several products (the reference and status).

### Expected behavior

The information displayed must be that of the current order return.

### Steps to reproduce

1 - Go to FO
2 - Order multiple different products
3 - Go to BO ( update order status set shipped)
4 - Go to SAV en enable order return feature
5 - Go to FO > My account > order history
6 - Return one product
7 - Return a second product  (NOTE : return the products one by one) 
8 - Go to FO > My account > Merchandise returns
9 - See the error : The reference and status of the first seconc (of first) is wrong

Capture d’écran du 2022-10-03 11-54-22
hargements/Capture d’écran du 2022-10-03 11-54-22.png

### PrestaShop version(s) where the bug happened

1.7.8.7, develop

### PHP version(s) where the bug happened

N/A

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by PululuK**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This change fixes minor code issues I found while working on another bug.
| Type?             | refacto
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/29839
| Related PRs       | N/A
| How to test?      | N/A
| Possible impacts? | N/A

### CODE REVIEW

**Comment by lartist** [comment]
Hi @PululuK,
Thanks for you PR 👍. Can you create a bug report to keep tracking on this bug?
Have a nice day

**Comment by PululuK** [comment]
> Hi @PululuK, Thanks for you PR +1. Can you create a bug report to keep tracking on this bug? Have a nice day

Hi @lartist, thanks, but is not realy a bug, just a minor code readability improvement, so I think no need issue.

**Comment by PululuK** [comment]
> 

Hi @lartist, check please https://github.com/PrestaShop/PrestaShop/issues/29839
Thanks

File: classes/order/OrderReturn.php [hunk 1]
```diff
@@ -146,7 +146,7 @@ public function countProduct()
         return (int) ($data['total']);
     }
 
-    public static function getOrdersReturn($customer_id, $order_id = false, $no_denied = false, Context $context = null)
+    public static function getOrdersReturn($customer_id, $order_id = false, $no_denied = false, Context $context = null, $idOrderReturn = null)
```
Comment (nicosomb) [inline classes/order/OrderReturn.php]: Why don't ask to have an `int` here?

Comment (PululuK) [reply classes/order/OrderReturn.php]: Yep, done ! Thanks @nicosomb

**Review by nicosomb — APPROVED** [review]

**Review by lartist — APPROVED** [review]

**Comment by lartist** [comment]
> > 
> 
> Hi @lartist, check please #29839 Thanks

Thanks @PululuK

**Review by AureRita — APPROVED** [review]
Hi @PululuK 

Thank you for your PR, I tested it and it works, as we can see here :

[Untitled_ Nov 29, 2022 5_41 PM.webm](https://user-images.githubusercontent.com/100699445/204590673-392a47b5-cf7e-44cc-9e57-7c3294defc7f.webm)

Thank you !

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @PululuK

**Pull request merged**